.travis.yml

```diff
@@ -21,7 +21,7 @@ before_script:
 - docker-compose -f docker-compose.yml -f docker-test.yml up -d
 - tail -f log/test.log &
 - until curl --silent -I -f --fail http://localhost:3000 ; do printf .; sleep 1; done
-- docker-compose exec api test/setup-parallel-env.sh
+- docker-compose exec -e TEST_RETRY_COUNT=3 api test/setup-parallel-env.sh
```

Comment:
@vasconsaurus , we should pass that environment variable in these `script` lines here, no?

https://github.com/meedan/check-api/blob/develop/.travis.yml#L47-L53

Reply:
yes! missed those, will update